### PR TITLE
issue #205: GitHub repo link

### PR DESCRIPTION
**Pull request by xoRmalka**
Resolving #205 

# Description

Mobile:
- Add a logo to the sidebar menu.

Mobile & Desktop:
- Add a GitHub icon that navigates to the repo link.
- Add dividers to the sidebar menu.

## screenshots
Desktop:

image

Mobile:
image

### CODE REVIEW

**Review by NoamGaash — APPROVED** [review]
love your style!
but - can you use i18n instead of the texts.tsx file?

**Comment by xoRmalka** [comment]
@NoamGaash 

Thank you!
About the i18n texts, done :)

**Review by NoamGaash — APPROVED** [review]

File: src/pages/components/header/sidebar/GitHubLink/GitHubLink.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  const data = {
+    label: t(TEXT_KEYS.github_link),
+    path: 'https://github.com/hasadna/open-bus-map-search',
+    icon: <GithubOutlined />,
+    element: null,
+  }
+
+  const handleClick = () => {
+    window.open(data.path, '_blank')
+  }
```
Comment (NoamGaash) [inline src/pages/components/header/sidebar/GitHubLink/GitHubLink.tsx:19]:
```suggestion
  const handleClick = useCallback(() => {
    window.open(data.path, '_blank')
  }, [])
````

**Pull request merged**